Makefile

```diff
@@ -51,6 +53,12 @@ build/install-plugins.sh:
 	curl -sSL $(SCRIPTS_URL)/install-plugins.sh > $@
 	chmod +x $@
 
+build/configuration-as-code:
+	git clone https://github.com/jenkinsci/configuration-as-code-plugin.git build/configuration-as-code
+
+build/configuration-as-code/target/configuration-as-code.hpi: build/configuration-as-code
+	cd build/configuration-as-code && mvn clean package -DskipTests
```

Comment:
If you're not planning to do anything special with the source, may as well just pull the .hpi from [J-o-J](https://ci.jenkins.io/job/Plugins/job/configuration-as-code-plugin/job/master/lastStableBuild/artifact/target/)?